securedrop_client/logic.py

```diff
@@ -401,6 +417,8 @@ def on_sync_success(self) -> None:
             * Download new messages and replies
             * Update missing files so that they can be re-downloaded
         """
+        self.set_status('')  # remove any permanent error status message
```

Comment:
should be `self.gui.clear_error_status`? it looks like `self.set_status` is modifying `TopPane.activity_status_bar` and `self.gui.clear_error_status` is modifying `TopPane.error_status_bar`

Reply:
fixed!

Reply:
ah so this should be `clear_error_status`, `update_error_status` will display the error status bar but with no message